crypto/signer.go

```diff
@@ -119,20 +123,21 @@ func NewStaticSigner(scheme Scheme, sig []byte, pubKey PublicKey) *StaticSigner
 	}
 }
 
-// Scheme returns neofscrypto.ECDSA_DETERMINISTIC_SHA256.
-// Implements neofscrypto.Signer.
+// Scheme returns the scheme that [StaticSigner] was instantiated with.
+// Implements [Signer].
 func (s *StaticSigner) Scheme() Scheme {
 	return s.scheme
 }
 
-// Sign returns precalculated hash.
-// Implements neofscrypto.Signer.
+// Sign returns precalculated serialized signature that was provided upon
+// [StaticSigner] creation. Never returns an error.
+// Implements [Signer].
 func (s *StaticSigner) Sign(_ []byte) ([]byte, error) {
 	return s.sig, nil
 }
 
-// Public returns neofscrypto.PublicKey.
-// Implements neofscrypto.Signer.
+// Public returns the key that [StaticSigner] was instantiated with.
```

Comment:
I suggest extending `the key that` to `the public key that`. I understand there are already many words _key_ and _public_, but in the current way looks like some inconsistency.

Reply:
Added.